public/app/features/logs/components/panel/virtualization.ts

```diff
@@ -6,8 +6,9 @@ import { LogListModel } from './processing';
 let ctx: CanvasRenderingContext2D | null = null;
 let gridSize = 8;
 let paddingBottom = gridSize * 0.75;
-let lineHeight = 22;
+export let lineHeight = 22;
```

Comment:
Maybe split this up into a local lineHeight variable to be mutated and export a const defaultLogRowLineHeight variable?

Reply:
Yeah, good call.